plugins/config.js

```diff
@@ -0,0 +1,42 @@
+export default function({ store }) {
+  // TODO: Figure out how to read this
+  // if (process.env.NODE_ENV !== 'production' && !('ACC' in global)) {
+  //   global.ACC = {
+  //     config: {}
+  //   }
+  // }
+
+  store.dispatch('config/init', {
```

Comment:
Should we only save debug settings that we use?

Reply:
What do you mean?

Reply:
Today you grab the whole copy of window.ACC.config and we only use 4 values from that file